preproc/Threshold_calibration_summary.ipynb

```diff
@@ -0,0 +1,1131 @@
+      "text/plain": [
+       "<Figure size 1080x504 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df_counts = process_run(run_number, nbits, field, tofpet_id, channels, plot=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33ea7faf",
+   "metadata": {},
+   "source": [
+    "**Add columns with data_time and expected_rate for vth_t2**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "ae605b7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_counts['data_time']     = (2**22 * 5) / 1e9 * df_counts['count']\n",
```

Comment:
Here it would be nice to add a comment explaining what these numbers represent. Since this notebook serves as an example, write all the comments you need to explain what is being done.

Reply:
Done!